README.md

```diff
@@ -4,3 +4,25 @@ SETUP:
 2: Seed database: npm run seed:db -> should result in 100 records inserted
 3: start server: npm start
 4: Build client pack: npm run build
+
+POSTGRES SETUP:
+DISCLAIMER - Postgres must be installed in your system for the following steps. Thank you.
+5: inside the database directory please create a file called "credentials.js"
+   inside credentials.js we will create a variable called initial and export the module.
+   initial should include your username, host, password, and port. Please see example below:
+    const initial = {
+      user: 'username',
```

Comment:
great job including instructions for user! I need to update my read me !! Later on, I recommend looking into using environment variables for this (checkout npm dotenv)

Reply:
alternatively/similarly is it possible to already have this file created with a //TO DO or template inside of it?

Reply:
oh! These are great, @claire-melbourne! If I end up moving forward with postgres I'll definitely implement these!